doc/Glossary/index.md

```diff
@@ -6,74 +6,239 @@ nav: /:path/Glossary/
-| Transport channel                                       | A Kaa client component responsible for communication with a Kaa server. Refer to Transports for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
```

Comment:
This is not true. If you are unsure about certain terminology, write smth like this:
Security token that is used during endpoint attachment procedure. Refer to [Endpoint Ownership] for more information.